pimd/pim_cmd.c

```diff
@@ -6512,7 +6512,7 @@ DEFUN(interface_no_ip_pim_boundary_oil,
 {
 	VTY_DECLVAR_CONTEXT(interface, iif);
 	struct pim_interface *pim_ifp;
-	int idx;
+	int idx = 4;
```

Comment:
I think one of the points of `argv_find()` is not having to worry about indexes, so the common idiom is to always initialize idx to zero. If we set idx manually then `argv_find()` wouldn't be necessary.

Reply:
I disagree with the statement that we initialize the idx to zero as a common idiom.  Go look through the code and we routinely set the value of the idx to what we think is the starting place to look in the argv array.

Reply:
I'll fix this